src/components/molecules/PollBox/PollBox.scss

```diff
@@ -0,0 +1,54 @@
+@import "scss/constants";
+
+.PollBox {
+  $spacing: 10px;
+  &__form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__section {
+    display: flex;
+    padding-bottom: $spacing;
+  }
+
+  &__input {
+    flex: 1;
+    input {
+      max-width: 305px;
+    }
```

Comment:
We tend not to style html elements like `input` directly. Is there a reason we need to set these styles on the `input` element rather than just within this class itself?

Also, where does the `305px` come from? Can we extract it as a variable + show any relevant calculations?

Reply:
It was fixed in the prev commits